scripts/docker-startup.sh

```diff
@@ -43,13 +43,11 @@ EOF
   osm2pgsql \
   --cache $OSM2PGSQL_CACHE \
   --number-processes $OSM2PGSQL_NUMPROC \
-  --hstore \
-  --multi-geometry \
   --database gis \
   --slim \
   --drop \
-  --style openstreetmap-carto.style \
-  --tag-transform-script openstreetmap-carto.lua \
+  --output flex \
+  --style openstreetmap-carto-flex.style \
```

Comment:
I have not tested the docker scripts (nor am i going to) - but this is the wrong file name.

Reply:
Fixed.